Main/Game.cs

```diff
@@ -416,6 +416,17 @@ public class Game {
                                 Map.currentLevel.metadata.hpDrain += key.Code == Keyboard.Key.Right ? 1 : -1;
                             }
                         }
+                        else if(key.Code == Keyboard.Key.F1 || key.Code == Keyboard.Key.F2) {
```

Comment:
Make it like 20ms without shift and 100ms with shift, because because of quantization the minimum value the PlayingOffset can change is 19-20ms and add key binds, that allow you to set the initial offset to the current offset and reset it (the current ones can be used for more fine tuning)